src/options/options.html

```diff
@@ -84,18 +97,7 @@ <h2 class="text-lg font-semibold text-gray-900">Select a workspace</h2>
             This workspace will be used to retrieve information about community members, and create new ones.
           </p>
 
-          <figure x-show="!!saveMessage" x-cloak role="alert"
-            class="flex gap-x-2 items-center pt-5 text-base font-semibold text-green-700">
-            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
-              stroke="currentColor" class="w-6 h-6">
-              <path stroke-linecap="round" stroke-linejoin="round"
-                d="M9.813 15.904L9 18.75l-.813-2.846a4.5 4.5 0 00-3.09-3.09L2.25 12l2.846-.813a4.5 4.5 0 003.09-3.09L9 5.25l.813 2.846a4.5 4.5 0 003.09 3.09L15.75 12l-2.846.813a4.5 4.5 0 00-3.09 3.09zM18.259 8.715L18 9.75l-.259-1.035a3.375 3.375 0 00-2.455-2.456L14.25 6l1.036-.259a3.375 3.375 0 002.455-2.456L18 2.25l.259 1.035a3.375 3.375 0 002.456 2.456L21.75 6l-1.035.259a3.375 3.375 0 00-2.456 2.456zM16.894 20.567L16.5 21.75l-.394-1.183a2.25 2.25 0 00-1.423-1.423L13.5 18.75l1.183-.394a2.25 2.25 0 001.423-1.423l.394-1.183.394 1.183a2.25 2.25 0 001.423 1.423l1.183.394-1.183.394a2.25 2.25 0 00-1.423 1.423z" />
-            </svg>
-
-            <figcaption x-text="saveMessage"></figcaption>
-          </figure>
-
-          <fieldset class="mt-5">
+          <fieldset class="mt-5" @change="disableSave = false">
```

Comment:
I think we can avoid the extra state variable here. We want to disable the save if the workspace selection is empty, so checking `x-bind:disable="!!selectedWorkspaceSlug"` might be enough to do the trick.

Reply:
Afraid not - `selectedWorkspaceSlug` gets set on click of "save". And if the save button is disabled, we can never achieve that.

We could set the selected workspace slug on change of the radio button selection, but then that is saved out of sync with the save button & with everything else, so might cause confusion.

Reply:
> selectedWorkspaceSlug gets set on click of "save"

I don’t believe so, it appears to be set by Alpine with the `x-model` directive on the radio button inputs. `save` takes that value and adds it to the storage. I gave it a go locally and it seems to work as intended (though it’s `x-bind:disable="!selectedWorkspaceSlug"`

Reply:
😱 Oh my gosh. You are 10000% right, thanks for calling that out - I have no *idea* what I was thinking. Too early, clearly. Unneeded state removed, working like a charm with your suggestions